src/Calendar/TableRow.tsx

```diff
@@ -88,7 +95,7 @@ class TableRow extends React.PureComponent<TableRowProps> {
   renderWeekNumber() {
     return (
       <div className={this.addPrefix('cell-week-number')}>
-        {format(this.props.weekendDate, 'W')}
+        {format(legacyParse(this.props.weekendDate), 'w')}
```

Comment:
iso week should be `I`
see: https://github.com/date-fns/date-fns/blob/master/CHANGELOG.md#200---2019-08-20

Reply:
Must be used in the local week because of this issue https://github.com/rsuite/rsuite/issues/892
If you add the props "IsoWeek", the weeks are correct https://www.epochconverter.com/weeks/2020 / http://myweb.ecu.edu/mccartyr/isowdcal.html

See attachments:
## Normal
<img width="595" alt="Screen Shot 2020-06-25 at 11 51 50" src="https://user-images.githubusercontent.com/26462869/85743447-6ad9b600-b6da-11ea-9b68-cd1d2ae46fc1.png">
## IsoWeek
<img width="597" alt="Screen Shot 2020-06-25 at 11 52 00" src="https://user-images.githubusercontent.com/26462869/85743463-6d3c1000-b6da-11ea-9b11-3197c6a355bb.png">